src/assets/PageNotFound/PageNotFound.css

```diff
@@ -0,0 +1,45 @@
+.not-found-container {
```

Comment:
kindly use module.css instead of plain css

Reply:
I have used module.css instead normal css file